tests/10-deploy.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import amulet
+import unittest
+
+seconds = 1100
+
+
+class TestPacketbeat(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """Setup the deployment for this class once and deploy."""
+        cls.deployment = amulet.Deployment(series='xenial')
```

Comment:
This doesn't appear to match the other tests, that are defined for trusty. I'm not going ot quibble over this, but thought i'd point it out.